tests/fixtures/wrangler_toml.rs

```diff
@@ -124,11 +124,15 @@ impl WranglerToml {
         account_id: &'static str,
         workers_dev: bool,
     ) -> WranglerToml {
-        WranglerToml {
+        let wrangler_toml = WranglerToml {
             workers_dev: Some(workers_dev),
             account_id: Some(account_id),
             ..WranglerToml::webpack(name)
-        }
+        };
+
+        eprintln!("{:#?}", &wrangler_toml);
```

Comment:
Debug statement?

Reply:
Yes to print the toml during tests, all the other toml helper functions had them except this one for some reason